File: ex03/ex03_04/ex03_04.py
def main():
  x = int(input("Enter x: "))
  shift = int(input("Enter number of digit to shift: "))

  new_x = x >> shift
  print(str(new_x) + " is right shift " + str(shift) + " digits of " + str(x))
  new_x = x << shift
  print(str(new_x) + " is left shift " + str(shift) + " digits of " + str(x))
  return 0

if __name__ == "__main__":
  main()

File: ex03/ex03_04/ex03_04.js
import promptSync from 'prompt-sync';
const prompt = promptSync();

function main() {
  let x = 0;
  let newX = 0;
  let shift = 0;

  x = parseInt(prompt("Enter x: "));
  shift = parseInt(prompt("Enter number of digit to shift: "));

  newX = x >> shift;
  console.log(String(newX) + " is right shift " + String(shift) + " digits of " + String(x));
  newX = x << shift;
  console.log(String(newX) + " is left shift " + String(shift) + " digits of " + String(x));

  return 0;
}

main();